dev/dev-vpc/terragrunt.hcl

```diff
@@ -0,0 +1,24 @@
+locals {
+  env_variables = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  environment   = local.env_variables.locals.environment
+
+}
+
+terraform {
+  source = "github.com/flugel-it/workshop-1-terraform-module"
+}
+
+include {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+    cidr_block       = "10.0.0.0/16"
+    instance_tenancy = "default"
+    vpc_id           = aws_vpc.main.id
```

Comment:
this one will not work, `aws_vpc.main.id` looks like a terraform resource, checked the [variables](https://github.com/flugel-it/workshop-vpc/blob/main/variables.tf) in the repository and this one is not expected as an input, you can remove the line